### DISCUSSION TITLE
Multiple pages on site are showing up as "404 file not found"

### DISCUSSION THREAD

**Discussion by manavbhatia** [community/community#23165] [Other Feature Feedback, Questions, & Ideas]
<p>Greetings! </p>
<p>I am seeing issues with opening multiple pages from my github pages site: <a href="https://mastmultiphysics.github.io" rel="nofollow noopener">https://mastmultiphysics.github.io</a></p>
<p>The repository is here: <a href="https://github.com/MASTmultiphysics/MASTmultiphysics.github.io" rel="nofollow noopener">https://github.com/MASTmultiphysics/MASTmultiphysics.github.io</a></p>
<p>The two pages that are showing the 404 error are: </p>
<p><a href="https://mastmultiphysics.github.io/_installation_page.html" rel="nofollow noopener">https://mastmultiphysics.github.io/_installation_page.html</a></p>
<p><a href="https://mastmultiphysics.github.io/_tutorials_page.html" rel="nofollow noopener">https://mastmultiphysics.github.io/_tutorials_page.html</a></p>
<p>However, these pages both exist in the repository. I have made several attempts to regenerate the site, but these specific pages always show up as “404 file not found”. </p>
<p>I would appreciate advice on this. </p>

**Comment by tdegeus** [comment, marked as answer]
<p>I had the same problem. GitHub help clarified this for me. This problem is caused by the jekyll build system that ignores files starting with an underscore. See:</p>
<p><a href="https://docs.github.com/en/github/working-with-github-pages/about-github-pages-and-jekyll#configuring-jekyll-in-your-github-pages-site" class="inline-onebox" rel="noopener nofollow ugc">About GitHub Pages and Jekyll - GitHub Docs</a>](<a href="https://docs.github.com/en/github/working-with-github-pages/about-github-pages-and-jekyll#configuring-jekyll-in-your-github-pages-site" class="inline-onebox" rel="noopener nofollow ugc">About GitHub Pages and Jekyll - GitHub Docs</a></p>
<p>The solution is to create a new empty file named <code>.nojekyll</code> in the root of your repository’s publishing source, which will bypass Jekyll.</p>
